.github/workflows/tag.yaml

```diff
@@ -0,0 +1,29 @@
+name: create_tag
+on: 
+  pull_request:
+    types: [closed]
+
+permissions:
+  contents: write
+
+jobs:
+  create_tag:
+    name: "Create tag"
+    if: github.event.pull_request.merged == true && contains(github.event.pull_request.labels.*.name, 'release-pr') && contains(github.event.pull_request.title, format('chore{0} Prepare', ':'))
```

Comment:
curious about the `{0}` variable, is there a reason for that?

Reply:
there is no way to escape `:` in string for workflow yaml in github. This is the only way I could find to format `chore: Prepare`.